python/python/benchmarks/test_scan.py

```diff
@@ -19,7 +19,7 @@
 import pyarrow.compute as pc
 import pytest
 
-NUM_ROWS = 1_000_000
+NUM_ROWS = 10_000
```

Comment:
Is this intentional?

Reply:
Yes, we made the rows much wider (>100KB), so in turn reduced the number of rows to compensate.